pgmoon/crypto.moon

```diff
@@ -2,9 +2,20 @@
 if ngx
   return { md5: ngx.md5 }
 
-crypto = require "crypto"
+digest = require "openssl.digest"
 
-md5 = (str) ->
-  crypto.digest "md5", str
+md5 = nil
+
+if digest
```

Comment:
This fallback won't works since requiring the module will error if it's not available

Reply:
Oh wow, yeah, I should have known that. That's definitely a "...what was I thinking" moment. 😳